templates/Admin/Tools/send_mail_to_user.html.twig

```diff
@@ -90,6 +105,16 @@
         <p>This tool allows you to send Emails to Users.</p>
 
         <div class="mtable">
+          <div class="row-title">
+            <label for="template-select">Template:</label>
+          </div>
+          <div class="drop-down">
+            <select id="template-select" name="template" class="drop-down-select">
+              <option value="0">Confirmation Email</option>
+              <option value="1">Reset Email</option>
+              <option value="2">Simple Message</option>
```

Comment:
using names instead of numbers makes the code more readable
```suggestion
              <option value="confirmation">Confirmation Email</option>
              <option value="reset">Reset Email</option>
              <option value="basic">Simple Message</option>
```